gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ManagedController.java

```diff
@@ -97,4 +103,23 @@ void passVetoMessage(
   List<String> getVetoRejections(@PathVariable("name") String name) {
     return keelService.getVetoRejections(name);
   }
+
+  @ExceptionHandler
+  void passthroughRetrofitErrors(RetrofitError e, HttpServletResponse response) {
+    try {
+      response.setStatus(e.getResponse().getStatus());
+      response.setHeader(
+          CONTENT_TYPE,
+          e.getResponse().getHeaders().stream()
+              .filter(it -> it.getName().equals(CONTENT_TYPE))
+              .map(Header::getValue)
+              .findFirst()
+              .orElse("text/plain"));
+      IOUtils.copy(e.getResponse().getBody().in(), response.getOutputStream());
+    } catch (Exception ex) {
+      log.error(
+          "Error reading response body when translating exception from downstream keelService: ",
+          ex);
```

Comment:
This is extraordinarily unlikely since we're reading raw bytes and not doing any parsing, but I wonder if we should/can handle this somehow?

Reply:
I have no idea what this exception would be or how to handle it. I'll handle it if it comes up in the future!